datarepo-clienttests/src/main/java/runner/TestRunner.java

```diff
@@ -434,29 +434,6 @@ void modifyHelmValuesAndDeploy() throws IOException {
     return modifiedEnvVars;
   }
 
-  void modifyKubernetesPostDeployment() throws Exception {
```

Comment:
I moved this method to the KubernetesClientUtils so it could be shared with the new scaling user journey.

Reply:
Agreed about pulling the code out to the utils class. Could you leave this method here though and just call out to your new utils method? The reason is that I think we'll want to modify other Kubernetes settings as part of the TestRunner setup work, they're just not implemented yet (e.g. pod memory size, auto-scale settings). So the idea is this modifyKubernetesPosDeployment will eventually call K8SClientUtils.scaleReplicaSetAndWaitToFinish, setAutoScale, setPodMemorySizeAndConfirm, etc.

Reply:
Ah, makes sense! I think I've made this change… let me know if it's not what you had in mind.

Reply:
Looks great, thank you!